### DISCUSSION TITLE
window:error page content

### DISCUSSION THREAD

**Discussion by GeorgianStan** [mdn/mdn-community#548] [Content]
The content of this [page](https://developer.mozilla.org/en-US/docs/Web/API/Window/error_event#event_type) seems to be different from the real results. 

**Event type**
Is mentioned that the [event type](https://developer.mozilla.org/en-US/docs/Web/API/Window/error_event#event_type) is `UIEvent`, however if we run this code we'll see in the console that the error is of type `ErrorEvent`. 
In [HTML spec](https://html.spec.whatwg.org/multipage/indices.html#event-error) is mentioned the same.
```js
window.addEventListener("error", (event) => {
  console.log(event); // event is of type ErrorEvent
});

const badCode = "const s;";
eval(badCode);
```

**Usage notes**
In [usage notes](https://developer.mozilla.org/en-US/docs/Web/API/Window/error_event#usage_notes) is mentioned:

> Unlike other events, the error event is canceled by returning true from the handler instead of returning false. When canceled, the error won't appear in the console, but the current script will still stop executing.

However, this is not true, regardless of the return statement `return true` or `return false` the error message is displayed in the console.
Here it looks like is similar with [unhandledrejection](https://developer.mozilla.org/en-US/docs/Web/API/Window/unhandledrejection_event#preventing_default_handling) and `event.preventDefault()` must be used so that the message is not printed to the console.

**Comment by schalkneethling** [comment]
Hey @GeorgianStan, thank you for starting the thread. From your description, I believe this might be a valid issue to open on the [content](https://github.com/mdn/content/issues/new/choose) repo.

**Reply by GeorgianStan** [reply]
[issue link](https://github.com/mdn/content/issues/24957)
